genesis/process/metaGenesisBlockCreator.go

```diff
@@ -65,6 +66,15 @@ func CreateMetaGenesisBlock(arg ArgsGenesisBlockCreator) (data.HeaderHandler, er
 		return nil, err
 	}
 
+	if arg.HardForkConfig.MustImport {
+		//TODO: think how to integrate when genesis is modified as well - how to integrate with imported data
+		// one example being - delete or not the validator statistics trie - and restart from 0, or should import from old
```

Comment:
Answer & discussed: recreate the validator statistics trie from the (newly) nodesSetup.json. Or a new predictable nodesSetup_1.json or something like this.

Reply:
would leave like this now, we can decide about this later.